packages/cw20/src/query.rs

```diff
@@ -84,3 +84,60 @@ pub struct AllAllowancesResponse {
 pub struct AllAccountsResponse {
     pub accounts: Vec<HumanAddr>,
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use cosmwasm_schema::schema_for;
+    use serde_json;
```

Comment:
Note we use a custom json serde lib for cosmwasm_std.

You can import cosmwasm_std::to_vec to use that. It would be good to ensure our custom lib supports flatten as well as the standard one